firebase-firestore/api.txt

```diff
@@ -149,8 +149,8 @@ package com.google.firebase.firestore {
 
   public class FirebaseFirestore {
     method @NonNull public com.google.firebase.firestore.ListenerRegistration addSnapshotsInSyncListener(@NonNull Runnable);
-    method @NonNull public com.google.firebase.firestore.ListenerRegistration addSnapshotsInSyncListener(Activity, @NonNull Runnable);
-    method @NonNull public com.google.firebase.firestore.ListenerRegistration addSnapshotsInSyncListener(Executor, @NonNull Runnable);
+    method @NonNull public com.google.firebase.firestore.ListenerRegistration addSnapshotsInSyncListener(@NonNull Activity, @NonNull Runnable);
```

Comment:
These changes are unrelated, and I thought I saw some other PR discussing resolving this. Please find that and make sure we're not making conflicting edits. If we revert this PR it shouldn't revert unrelated APIs.